config/post-meta.json

```diff
@@ -1 +1,24 @@
-{}
\ No newline at end of file
+{
+  "fm_gutenberg_open_graph_description": {
```

Comment:
This update should be reverted - postmeta for this plugin should be dynamically defined in user code vs. being set in this config. Ideally, this config file would be removed and the code that calls it would also be removed, in favor of just calling `register_meta_helper` dynamically when fields are registered.